test/ConfigManagerTest.cpp

```diff
@@ -341,18 +340,133 @@ void checkValidator(ConfigManager& manager,
       ::testing::ContainsRegex(containedInExpectedErrorMessage));
 }
 
-TEST(ConfigManagerTest, ConfigOptionWithValidator) {
-  ConfigManager managerWithoutSubManager{};
-  int notUsedInt;
-  managerWithoutSubManager.addOption("h", "", &notUsedInt)
-      .getConfigOption()
-      .addValidator([](const int& n) { return 0 <= n && n <= 24; },
-                    "Validator of h");
-  checkValidator(managerWithoutSubManager,
-                 nlohmann::json::parse(R"--({"h" : 10})--"),
-                 nlohmann::json::parse(R"--({"h" : 100})--"), "Validator of h");
+// Human readable examples for `addValidator`.
+TEST(ConfigManagerTest, HumanReadableAddValidator) {
+  ConfigManager m{};
+
+  // The number of the option should be in a range. We define the range by using
+  // two validators.
+  int someInt;
+  decltype(auto) numberInRangeOption =
+      m.addOption("numberInRange", "", &someInt);
+  m.addValidator([](const int& num) { return num <= 100; },
+                 "'numberInRange' is bigger than 100.", numberInRangeOption);
```

Comment:
I would always use positive formulations like `must be`, so 
`numberInRange must be <= 100`. (That's what we also should do in "actual" code).

Reply:
Thank you for the advice! Implemented in [ce5b0fb](https://github.com/ad-freiburg/qlever/pull/1058/commits/ce5b0fba5625757bf24e173c4cd7010d832b1e69).